### PR TITLE
86b1vpzga

### PR DESCRIPTION
**Pull request by Vihari7**
- Added a seeding script using Prisma to populate the user table with mentors and students.
- Integrated bcrypt to hash default passwords for security purposes.
- Included a symmetric key for consistent password hashing.
- Set-up mentor-student relationships in the data to create a realistic initial setup.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (bsaranga) [inline package.json]: What's this file?

File: server/tsconfig.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
     "allowJs": true,
     "skipLibCheck": true,
     "strict": true,
-    "noEmit": true,
+    "noEmit":false,
     "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "bundler",
+    "module": "commonjs",
```
Comment (bsaranga) [inline server/tsconfig.json]: This might impact other imports in the application. Please run the application and see if everything is working as expected.

**Review by bsaranga — CHANGES_REQUESTED** [review]
Looks like there are some files that aren't directly related to your task. Make sure you take a pull from `development` and resolve any merge conflicts.

File: server/tsconfig.json (hunk 2 above)
Comment (Vihari7) [reply server/tsconfig.json]: Sir, I made the corrections and run the application again. Everything works as expected. please let me know if there's anything else to check.

File: node_modules/.package-lock.json
Comment (bsaranga) [inline node_modules/.package-lock.json:1]: delete this

**Review by bsaranga — CHANGES_REQUESTED** [review]
Please pull from `development` branch and resolve the conflicts.

File: package-lock.json
Comment (bsaranga) [inline package-lock.json:1]: Delete this file, we don't want a root package-lock

File: server/tsconfig.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "allowJs": true,
     "skipLibCheck": true,
     "strict": true,
-    "noEmit": true,
+    "noEmit":true,
```
Comment (bsaranga) [inline server/tsconfig.json]: Revert this by keeping a space between the : and true.

**Review by bsaranga — CHANGES_REQUESTED** [review]

File: server/package.json [hunk 4]
```diff
@@ -28,6 +30,7 @@
     "dotenv-cli": "^7.4.2",
     "lucide-react": "^0.436.0",
     "next": "14.2.5",
+    "npm": "^10.8.2",
```
Comment (bsaranga) [inline server/package.json]: This should not be an application dependency. Remove it.

File: server/package.json [hunk 5]
```diff
@@ -49,7 +53,9 @@
     "eslint-config-next": "14.2.5",
     "jest": "^29.7.0",
     "postcss": "^8",
+    "prisma": "^5.18.0",
     "tailwindcss": "^3.4.1",
+    "ts-node": "^10.9.2",
```
Comment (bsaranga) [inline server/package.json]: Duplicate `ts-node` entry:

Remove that line manually.

File: server/package.json [hunk 6]
```diff
@@ -28,6 +30,7 @@
     "dotenv-cli": "^7.4.2",
     "lucide-react": "^0.436.0",
     "next": "14.2.5",
+    "npm": "^10.8.2",
     "prisma": "^5.18.0",
```
Comment (bsaranga) [inline server/package.json]: Remove prisma from `dependencies`, it should only be a `devDependency`.

**Review by bsaranga — CHANGES_REQUESTED** [review]
Attend to the changes requested.

**Review by bsaranga — APPROVED** [review]

**Pull request merged**